flask-deployment.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: todo-app-deployment
+  labels:
+    app: todo-app
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: todo-app
+  template:
+    metadata:
+      labels:
+        app: todo-app
+    spec:
+      containers:
+        - name: todo-container
+          image: todo-app:latest
+          imagePullPolicy: Never
```

Comment:
@Arun07L why do we have Never as the image pull policy?

Reply:
local registry la irrunthu image ah pull panni edukum pothu suppose image illana pull pantra attempt edukathu.